lighthouse-core/test/lib/url-shim-test.js

```diff
@@ -29,6 +29,12 @@ describe('URL Shim', () => {
     assert.doesNotThrow(_ => new URL(url));
   });
 
+  it.skip('handles URLs with multiple dashes', () => {
+    // from https://github.com/GoogleChrome/lighthouse/issues/1972
+    const url = 'https://r15---sn-o097znl7.googlevideo.com/generate_204?conn2';
+    assert.doesNotThrow(_ => new URL(url));
```

Comment:
ha, fair. I guess it depends on what the purpose of the test is. In this case where you're not testing the output I guess it makes some sense to show what you are testing...but there seems to be a fine line between that and the fact that every test that isn't running `assert.throws()` is implicitly asserting that it isn't going to throw.

I guess in this case the regression is that the constructor was throwing, not that it was parsing incorrectly, so I guess it's not the worst thing to be explicit about that with the assertion statement.